integration/test_auth.py

```diff
@@ -219,6 +237,78 @@ def test_get_user(new_user_with_params):
     provider_ids = sorted([provider.provider_id for provider in user.provider_data])
     assert provider_ids == ['password', 'phone']
 
+class TestGetUsers:
+    @staticmethod
+    def _map_user_record_to_uid_email_phones(user_record):
+        return {
+            'uid': user_record.uid,
+            'email': user_record.email,
+            'phone_number': user_record.phone_number
+        }
+
+    def test_returns_users_by_various_identifier_types_in_a_single_call(self, new_user_record_list):
```

Comment:
Use shorter names to be consistent with the rest of this file?

`test_multiple_uid_types():`